bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToZATo2L2B_ggfusion_b-associatedproduction/HToZATo2L2B_1000p00_200p00_0p50_bbH4F_TuneCP5_13TeV-amcatnloFXFX-pythia8/HToZATo2L2B_1000p00_200p00_0p50_bbH4F_TuneCP5_13TeV-amcatnloFXFX-pythia8_run_card.dat

```diff
@@ -0,0 +1,196 @@
+              ! numbers for central PDF sets are allowed. Can be a list; 
+              ! PDF sets beyond the first are included via reweighting.
+#***********************************************************************
+# Include the NLO Monte Carlo subtr. terms for the following parton    *
+# shower (HERWIG6 | HERWIGPP | PYTHIA6Q | PYTHIA6PT | PYTHIA8)         *
+# WARNING: PYTHIA6PT works only for processes without FSR!!!!          *
+#***********************************************************************
+  PYTHIA8	= parton_shower 
+  1.0	= shower_scale_factor ! multiply default shower starting
+                                  ! scale by this factor
+#***********************************************************************
+# Renormalization and factorization scales                             *
+# (Default functional form for the non-fixed scales is the sum of      *
+# the transverse masses divided by two of all final state particles    * 
+# and partons. This can be changed in SubProcesses/set_scales.f or via *
+# dynamical_scale_choice option)                                       *
+#***********************************************************************
+  False	= fixed_ren_scale ! if .true. use fixed ren scale
+  False	= fixed_fac_scale ! if .true. use fixed fac scale
+  -1	= dynamical_scale_choice ! Choose one (or more) of the predefined
+           ! dynamical choices. Can be a list; scale choices beyond the
+           ! first are included via reweighting
+  91.118  = mur_ref_fixed    ! fixed ren reference scale
+  91.118  = muf_ref_fixed ! fixed fact reference scale
+  1.0   = mur_over_ref ! ratio of current muR over reference muR
+  1.0   = muf_over_ref ! ratio of current muF over reference muF 
+#*********************************************************************** 
+# Reweight variables for scale dependence and PDF uncertainty          *
+#***********************************************************************
+  1.0, 2.0, 0.5	= rw_rscale ! muR factors to be included by reweighting
+  1.0, 2.0, 0.5	= rw_fscale ! muF factors to be included by reweighting
+  True	= reweight_scale ! Reweight to get scale variation using the 
+            ! rw_rscale and rw_fscale factors. Should be a list of 
+            ! booleans of equal length to dynamical_scale_choice to
+            ! specify for which choice to include scale dependence.
+  False	= reweight_pdf ! Reweight to get PDF uncertainty. Should be a
+            ! list booleans of equal length to LHAPDF ID to specify for
+            !  which PDF set to include the uncertainties.
+  -1.0	= rw_fscale_down  ! lower bound for fact scale variations
+  -1.0	= rw_rscale_down  ! upper bound for fact scale variations
+  -1.0	= rw_fscale_up   ! upper bound for fact scale variations
+  -1.0	= rw_rscale_up   ! upper bound for ren scale variations
+  325501   = PDF_set_min      ! First of the error PDF sets 
+  325600   = PDF_set_max      ! Last of the error PDF sets
```

Comment:
We don't use this logic but the systematics module in rumcmsgrid_NLO.sh. Please remove.